pt/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: news_post
+title: "Lançado o Ruby 2.4.0-preview2"
+author: "naruse"
+translator: "Filipe Rocha"
+date: 2016-09-08 18:00:00 +0900
+lang: pt
+---
+
+Temos o gosto de anunciar a versão 2.4.0-preview2 do Ruby.
+
+A versão 2.4.0-preview2 do Ruby é a segunda *preview* do Ruby 2.4.0,
+lançada com o fim de obter *feedback* da comunidade.
+Poderão
+[enviar *feedback*](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)
+uma vez que ainda podem influenciar das funcionalidades.
+
+## [Unificar as classes Fixnum e Bignum na classe Integer](https://bugs.ruby-lang.org/issues/12005)
+
+Apesar do [ISO/IEC 30170:2012](http://www.iso.org/iso/iso_catalogue/catalogue_tc/catalogue_detail.htm?csnumber=59579)
+não especificar os detalhes da classe Integer,
+o Ruby teve duas classes Integer visíveis: Fixnum and Bignum.
+O Ruby 2.4 unifica-as na classe Integer.
+Todas as extensões C que utilizam a classe Fixnum ou Bignum necessitam de ser corrigidas.
+
+Consultar também [o ticket](https://bugs.ruby-lang.org/issues/12005) e os [slides do akr](http://www.a-k-r.org/pub/2016-09-08-rubykaigi-unified-integer.pdf).
+
+## [String suporta Unicode case mappings](https://bugs.ruby-lang.org/issues/10085)
+
+`String/Symbol#upcase/downcase/swapcase/capitalize(!)` manipula, agora,
+maiúsculas e minúsculas em Unicode em vez de o fazer somente em ASCII.
+
+## Melhorias de desempenho
+
+O Ruby 2.4 contém, também, as seguintes melhorias na performance incluindo melhorias na linguagem:
+
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
+
+`[x, y].max` e `[x, y].min` estão optimizados para evitar a criação de um array temporário em certas condições.
+
+### [Regexp#match?](https://bugs.ruby-lang.org/issues/8110)
+
+Adicionado `Regexp#match?`, que executa um *regexp match* sem a criação
+de um objecto de referência e alterando `$~` para reduzir a alocação de objectos.
+
+### Outras melhorias de desempenho
+
+* [acelerar o acesso às variáveis de instância](https://bugs.ruby-lang.org/issues/12274)
+
+## Depuração
+
+### [Thread#report_on_exception and Thread.report_on_exception](https://bugs.ruby-lang.org/issues/6647)
+
+O Ruby ignora exceções nas *threads*, a não ser que outra *thread* se junte a elas.
+Utilizando `report_on_exception = true`,
+poderá verificar se uma *thread* terminou devido a uma exceção não tratada.
```

Comment:
I think `é possível` looks nicer instead of `poderá`, but again, Brazilian. 😄 

And I belive, if I'm not mistaken, it's `devido à`.